rll/setup_automatic_upgrade.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
All updates done in [IV-957_iteration_rl10_upgrade branch] (https://github.com/rightscale/rightlink_scripts/tree/IV-957_iteration_rl10_upgrade)